src/microbit/pxt_vis_navnet_ditt/pxt_vis_navnet_ditt.md

```diff
@@ -0,0 +1,57 @@
+---
+title: "PXT: Vis navnet ditt"
+author: Helene Isnes
+language: nb
+---
+
+
+# Introduksjon {.intro}
+I denne introduksjonsoppgaven skal vi få microbiten til å vise navnet ditt. 
+
+
+# Steg 1: Vi finner "ved start"-klossen {.activity}
+
+## Sjekkliste {.check}
+
+- [ ] Start et nytt PXT-prosjekt, for eksempel ved å gå til 
+  [makecode.microbit.org](https://makecode.microbit.org/?lang=no).
+
+- [ ] Klikk på gruppa `Basis` som ligger øvert i menyen (midt på skjermen). 
+	Finn klossen som heter `ved start`(den ligger nederst) og klikk på den.
+
+![Bilde av ved start klossen](ved_start.png)
+
+- [ ] Den skal nå ha dukket opp i kodefeltet ditt. Du kan flytte den hvor du vil i feltet; det har ikke noe å si hvor den ligger hen. (Hvis du allerede hadde den fra før, slett en av den ved å dra den bort til menytfeltet og slipp.)
```

Comment:
et nytt prosjekt starter vel alltid med to kodeblokker? "ved start" og "gjenta for alltid". muligens skrive om denne kommentaren i parentes.